Makefile

```diff
@@ -185,28 +184,57 @@ container-test:
 		-t $(DOCKER_REPO):local_e2e_test .
 endif
 
-.PHONY: container
-container: Dockerfile
-	$(OCI_BIN) build --build-arg BUILD_DATE="$(BUILD_TIMESTAMP)" \
+.PHONY: container-build
+container-build:
+	git update-index --refresh
```

Comment:
Should this line be commented? Going by https://github.com/observatorium/thanos-receive-controller/blob/master/Makefile#L97?

Reply:
Ugh, that should be uncommented over there. 👀

Reply:
Opened a PR to fix the controller: https://github.com/observatorium/thanos-receive-controller/pull/117